src/util/createStoreServer.js

```diff
@@ -2,7 +2,7 @@ import { applyMiddleware, createStore } from 'redux';
 import { parseQueryResponse } from './fetchUtils';
 import getEpicMiddleware from '../middleware/epic';
 import catchMiddleware from '../middleware/catch';
-import apiProxy$ from '../apiProxy/api-proxy';
+import 'rxjs/add/operator/map';
```

Comment:
it was always using the RxJS map, but we were missing the explicit operator import - it just happened to still work because a parent dependency must have also added the operator. This import just enforces the dependency properly.